### DISCUSSION TITLE
自动部署 在什么时候会触发？

### DISCUSSION THREAD

**Discussion by losie** [Qexo/Qexo#498] [General]
![image](https://github.com/Qexo/Qexo/assets/36534236/0b892750-f843-459c-8e9b-2aa6752170a7)
修改源码还是 怎么的

**Comment by losie** [comment]
请问一下 大佬们！

**Comment by am-abudu** [comment]
触发save或delete函数（可以理解为在Qexo修改源码），并且autobuild配置存在时触发
